src/config/configuration.ts

```diff
@@ -22,10 +22,26 @@ const configuration = () => {
   const proposalGroups = process.env.PROPOSAL_GROUPS || ("" as string);
   const sampleGroups = process.env.SAMPLE_GROUPS || ("" as string);
 
+  const defaulgLogger = {
+    type: "DefaultLogger",
+    modulePath: "./loggingProviders/defaultLogger",
+    config: {},
+  };
+  const jsonConfigMap: { [key: string]: object[] } = {};
+  const jsonConfigFileList = ["loggers"];
+  for (const fileName of jsonConfigFileList) {
+    const filePath = `${fileName}.json`;
+    if (fs.existsSync(filePath)) {
+      const data = fs.readFileSync(filePath, "utf8");
+      jsonConfigMap[fileName] = JSON.parse(data);
+      break;
+    }
+  }
```

Comment:
I would make it more generic, so we can easily use it for the functional accounts and future development.
How about something like this:
```typescript
  const jsonConfigFileList = {
    "loggers" : process.env.LOGGERS_CONFIG_FILE || "loggers.json",
  };
 Object.keys(jsonConfigFileList).forEach( key => {
    const filePath = jsonConfigFileList[key];
    if (fs.existsSync(filePath)) {
      const data = fs.readFileSync(filePath, "utf8");
      jsonConfigMap[fileName] = JSON.parse(data);
      break;
    }
  });
```